README.md

```diff
@@ -30,25 +31,39 @@ Next, tell OmniAuth about this provider. For a Rails app, your `config/initializ
 
 ```ruby
 Rails.application.config.middleware.use OmniAuth::Builder do
-  provider :slack, "API_KEY", "API_SECRET", scope: "client"
+  provider :slack, 'API_KEY', 'API_SECRET', scope: 'identity.basic'
 end
 ```
 
-Replace `"API_KEY"` and `"API_SECRET"` with the appropriate values you obtained [earlier](https://api.slack.com/applications).
+Replace `'API_KEY'` and `'API_SECRET'` with the appropriate values you obtained [earlier](https://api.slack.com/applications).
 
 If you are using [Devise](https://github.com/plataformatec/devise) then it will look like this:
 
 ```ruby
 Devise.setup do |config|
   # other stuff...
 
-  config.omniauth :slack, ENV["SLACK_APP_ID"], ENV["SLACK_APP_SECRET"], scope: 'client'
+  config.omniauth :slack, ENV['SLACK_APP_ID'], ENV['SLACK_APP_SECRET'], scope: 'identity.basic'
 
   # other stuff...
 end
 ```
 
-Slack lets you choose from a [few different scopes](https://api.slack.com/docs/oauth#auth_scopes).
+
+## Scopes
+Slack lets you choose from a [few different scopes](https://api.slack.com/docs/oauth-scopes#scopes).
+
+However, you cannot request both `identity` scopes and other scopes at the same time.
+
+If you need to combine regular app scopes with those used for “Sign in with Slack”, you should
+configure two providers:
+
+```ruby
+provider :slack, 'API_KEY', 'API_SECRET', scope: 'identity.basic', name: :sign_in_with_slack
+provider :slack, 'API_KEY', 'API_SECRET', scope: 'team:read,users:read,identify,bot'
```

Comment:
@capripot the first one is actually named `:sign_in_with_slack`. Notice the last argument. That's the name you'll need to use to refer to it.

`:slack` is the name of the strategy class to be used